database_schemas/cassandra/benchflow.cql

```diff
@@ -184,24 +183,24 @@ CREATE TABLE host_properties (
 
 -- Experiment metrics
 CREATE TABLE exp_process_duration (
-  process_duration_weighted_avg float,
-  process_duration_q1_min float,
-  process_duration_q1_max float,
-  process_duration_q2_min float,
-  process_duration_q2_max float,
-  process_duration_q3_min float,
-  process_duration_q3_max float,
-  process_duration_p95_min float,
-  process_duration_p95_max float,
-  process_duration_ci095_min float,
-  process_duration_ci095_max float,
-  process_duration_median_min float,
-  process_duration_median_max float,
-  process_duration_mode_min float,
-  process_duration_mode_max float,
-  process_duration_min float,
-  process_duration_max float,
-  process_duration_sd float,
+  process_duration_weighted_avg double,
+  process_duration_q1_min bigint,
```

Comment:
A wanted to say `double`. Basically it can have decimal precision, `bigint` it is not suitable for that.